src/backend/distributed/planner/fast_path_router_planner.c

```diff
@@ -150,27 +156,73 @@ GeneratePlaceHolderPlannedStmt(Query *parse)
 }
 
 
+static void
+InitializeFastPathContext(FastPathRestrictionContext *fastPathContext,
```

Comment:
just to remind the point we discussed - we can consider adding a function comment for this:

```c
/*
 * InitializeFastPathContext..
 */
InitializeFastPathContext(..
```